### ISSUE TITLE
Possible to lint vue files via CLI?

### ISSUE THREAD

**Issue by gjabell** [johnsoncodehk/tsslint#31]
Hi! I found this project the other day and I would love to use it as an alternative to `typescript-eslint` for the typed lintings, as they are currently very slow with eslint.

I created the following config based on reading some of the other issues:

```typescript
import { defineConfig } from '@tsslint/config'
import { convertConfig } from '@tsslint/eslint'

export default defineConfig({
  rules: convertConfig({
    '@typescript-eslint/no-misused-promises': 'error',
    '@typescript-eslint/no-floating-promises': 'error',
  }),
})
```

Running the CLI is very much faster than eslint however it only shows errors in `.ts` files, despite Vue files being included in the config:

```json
{
  "includes": ["src/**/*.ts", "src/**/*.vue"]
}
```

After some investigation it seems that the culprit is [this call here](https://github.com/johnsoncodehk/tsslint/blob/master/packages/cli/index.ts#L321), as by default the `fileNames` returned only include TypeScript files. I hacked around a bit and was able to get it to recognize Vue files as well with the following change:

```diff
- return ts.parseJsonSourceFileConfigFileContent(jsonConfigFile, ts.sys, path.dirname(tsconfig), {}, tsconfig);
+ return ts.parseJsonSourceFileConfigFileContent(jsonConfigFile, ts.sys, path.dirname(tsconfig), {}, tsconfig, undefined, [{ extension: 'vue', isMixedContent: true, scriptKind: 7 }]);
```

but unfortunately it fails once it reaches a Vue file with:

```
▲  Something went wrong
```

What's interesting is if I use the typescript-language-server plugin, the `<script lang="ts">` portion of the Vue files are correctly linted and will show errors according to the config.

As I was hoping to use this as a faster linter for CI as well as locally, is there any way to apply the same checks on non-TypeScript files like it works within the language server?

Thanks!

**Comment by johnsoncodehk** [comment]
It didn't come true because you were the first to ask for it.

Vue support is available naturally in the IDE (as long as hybrid mode is enabled). In the CLI we need to integrate `@vue/language-core`, I'm looking into it.

**Comment by johnsoncodehk** [comment]
Please track #32

**Comment by johnsoncodehk** [comment]
@gjabell it should be support in 1.4.1, could you take try? Thanks.

**Comment by johnsoncodehk** [comment]
@gjabell thanks for the reproduction, this problem should be fixed in 1.4.2.
